### PR TITLE
MSPSDS-350: Set up prod environment

### PR DESCRIPTION
**Pull request by jasiekmiko**
Some changes will affect all environments:
- move from AWS to PaaS elasticsearch
- remove basic HTTP

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -67,9 +67,9 @@ jobs:
   - name: Deploy keycloak
     if: branch = staging AND type != pull_request
     script: BRANCH=${TRAVIS_BRANCH} BUILD_ID=${TRAVIS_BUILD_NUMBER} SPACE=staging COMPONENT=keycloak ./ci/deploy-if-changed.sh
-  - name: Deploy cosmetics
-    if: branch = staging AND type != pull_request
-    script: BRANCH=${TRAVIS_BRANCH} BUILD_ID=${TRAVIS_BUILD_NUMBER} SPACE=staging COMPONENT=cosmetics-web ./ci/deploy-if-changed.sh
+#  - name: Deploy cosmetics
```
Comment (jasiekmiko) [inline .travis.yml]: @DWRendell FYI I've commented out these steps while those envs don't exist yet - let me know if that's troublesome :)

Comment (DWRendell) [reply .travis.yml]: Brill, thanks :)

File: web/config/initializers/elasticsearch.rb [hunk 2]
```diff
@@ -1,21 +1 @@
-require "faraday_middleware/aws_sigv4"
-
-def aws_elasticsearch_client
-  Elasticsearch::Client.new(url: ENV["AWS_ELASTICSEARCH_URL"]) do |f|
```
Comment (broder) [inline web/config/initializers/elasticsearch.rb]: We should update the AWS environment variables / access keys to not include any elasticsearch stuff once this is merged.

**Comment by jasiekmiko** [comment]
thanks @broder :)

**Pull request merged**